src/components/toolbar/AppUserMenu.tsx

```diff
@@ -22,6 +22,7 @@ import { useQuickConnectEnabled } from 'hooks/useQuickConnect';
 import globalize from 'lib/globalize';
 import shell from 'scripts/shell';
 import Dashboard from 'utils/dashboard';
+import { Download } from '@mui/icons-material';
```

Comment:
Move this up (after `import DashboardIcon from '@mui/icons-material/Dashboard';`) and use full path to help tree-shaking.